tests/test_uint256_checked.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import asyncio
+from starkware.starknet.testing.starknet import Starknet
+from utils import uint, MAX_UINT256, assert_revert, add_uint, sub_uint
+
+
+@pytest.fixture(scope='module')
+def event_loop():
+    return asyncio.new_event_loop()
+
+
+@pytest.fixture(scope='module')
+async def uint256_checked_factory():
+    starknet = await Starknet.empty()
+    uint256_checked = await starknet.deploy(
+        "contracts/utils/uint256_checked_test.cairo"
+    )
+
+    return uint256_checked
+
+
+@pytest.mark.asyncio
+async def test_add(uint256_checked_factory):
+    uint256_checked = uint256_checked_factory
+
+    a = (5678, 9)
+    b = (123, 4)
+    c = add_uint(a, b)
```

Comment:
don't we have an `uint()` interface already in `utils.py`? we should be consistent so whenever we change anything in it, it propagates across the codebase